transform/mattermost-analytics/models/intermediate/product/nps/int_nps_server_version_spined.sql

```diff
@@ -0,0 +1,27 @@
+WITH first_server_version AS (
+    SELECT 
+        server_id,
+        server_version,
+        MIN(event_date) AS min_event_date
+    FROM {{ ref('int_nps_score') }}
+    WHERE event_date >= '{{ var('nps_start_date')}}'
+    GROUP BY all
```

Comment:
Given that group by is happening on server_id, server_version, multiple rows for the same server_id, server_version will exist in the CTE. Shouldn't the first server date be unique for each server?

Reply:
Removed the `server_version` from this and reverted it to the way it was before.